modules/quantcastIdSystem.js

```diff
@@ -7,11 +7,153 @@
 
 import {submodule} from '../src/hook.js'
 import { getStorageManager } from '../src/storageManager.js';
+import { triggerPixel, logInfo } from '../src/utils.js';
+import { uspDataHandler, coppaDataHandler, gdprDataHandler } from '../src/adapterManager.js';
 
 const QUANTCAST_FPA = '__qca';
+const DEFAULT_COOKIE_EXP_TIME = 392; // (13 months - 2 days)
+const PREBID_PCODE = 'p-KceJUEvXN48CE'; // Not associated with a real account
+const DOMAIN_QSERVE = 'https://pixel.quantserve.com/pixel';
```

Comment:
`QSERVE_URL` ? This isn't really a domain.

Reply:
_Question_: Do we want to use a protocol relative URL instead? I mean maybe not these days, but it is common practice.
_Note_: If we did, it would be `QSERVE_URL = "//pixel.quantserve.com/pixel";`

Reply:
Hmm, the advantage of this is that the URL would have the same protocol as the site it is on (versus always being https), right? I think that makes sense. quantJS also follows that.

Reply:
i think we need to use secure these days for certain cookie based stuff to work in Chrome? (might be mistaken about that).  I guess that's less of a concern here, although i suppose these requests will set 3rd party cookies where they can.

My vote would just be to keep this secure i think.

We should use `QSERVE_URL` as Scott suggests though.